v3/python/basic_list_apks_service_account.py

```diff
@@ -21,6 +21,7 @@
 from apiclient.discovery import build
 import httplib2
 from oauth2client import client
+from oauth2client.service_account import ServiceAccountCredentials
```

Comment:
I think you forgot to actually use it. You are still using `SignedJwtAssertionCredentials`.

Reply:
Also, `oauth2client` is deprecated now, so I would probably completely replace this sample with one using the `Credentials` from `google.oauth2` instead.

NOTE: I am not part of the Google community, just sharing my opinion

Reply:
> 
> 
> I think you forgot to actually use it. You are still using `SignedJwtAssertionCredentials`.

You're right! I overlooked that.

> 
> 
> Also, `oauth2client` is deprecated now, so I would probably completely replace this sample with one using the `Credentials` from `google.oauth2` instead.
> 
> NOTE: I am not part of the Google community, just sharing my opinion

Yeah, I'll leave it open to Google and on how they decide to proceed with the samples. Good to have your comment here though for others to read.